live-demo/live-demo.js

```diff
@@ -37,12 +37,20 @@ function complex () {
       { name: 'sex', value: 'Male', type: 'radio' },
       { name: 'sex', value: 'Female', type: 'radio' },
 
-      { name: 'skills', value: 'JS', type: 'checkbox' },
-      { name: 'skills', value: 'Ruby', type: 'checkbox' },
-      { name: 'skills', value: 'Java', type: 'checkbox' }
-    ]
-  }, function (isConfirm) {
-    // do whatever you want with the form data
-    console.log(this.swalForm) // { name: 'user name', nickname: 'what the user sends' }
+          { name: 'skills', value: 'JS', type: 'checkbox' },
+          { name: 'skills', value: 'Ruby', type: 'checkbox' },
+          { name: 'skills', value: 'Java', type: 'checkbox' },
+
+          { id: 'select', type: 'select',options: [
+            {value:"test1",text: "test1"},
+            {value:"test2",text: "test2"},
+            {value:"test3",text: "test3"},
+            {value:"test4",text: "test4"},
+            {value:"test5",text: "test5"}
+          ]},
+      ]
+  }, function(isConfirm) {
+      // do whatever you want with the form data
+      console.log(this.swalForm); // { name: 'user name', nickname: 'what the user sends' }
```

Comment:
Please review feross/standard compliance. This particular line for example is not compliant. To check compliance you need to first install feross/standard using `npm install standard` and then run the install command (everything within project's directory)/